packages/account/src/providers/provider.ts

```diff
@@ -1741,6 +1741,24 @@ export default class Provider {
     return { transactions, pageInfo };
   }
 
+  /**
+   * Fetches a compressed block at the specified height.
+   *
+   * @param height - The height of the block to fetch.
+   * @returns The compressed block if available, otherwise `null`.
+   */
+  async daCompressedBlock(height: string) {
```

Comment:
Why did you add this method? `this.operations.daCompressedBlock` seems enough.

Reply:
I don't have a specific reason for it. I implemented it simply because it might be useful for users.